decidim-initiatives/spec/types/integration_schema_spec.rb

```diff
@@ -38,7 +38,7 @@
         "id" => initiative.type.id.to_s,
         "initiatives" => initiative.type.initiatives.map { |i| { "id" => i.id.to_s } },
         "minimumCommitteeMembers" => initiative.type.minimum_committee_members,
-        "promotingComitteeEnabled" => initiative.type.promoting_committee_enabled,
+        "promotingCommitteeEnabled" => initiative.type.promoting_committee_enabled,
```

Comment:
this might require more stuff

Reply:
no need. This is a spec, and the initial change has been handled in the changes of decidim-initiatives/lib/decidim/api/initiative_api_type.rb